docs/content/commands/npm-logout.md

```diff
@@ -10,6 +10,8 @@ description: Log out of the registry
 npm logout [--registry=<url>] [--scope=<@scope>]
 ```
 
+Note: This command is unaware of workspaces.
```

Comment:
same comment as adduser (which, since this is called "logout", should really have "login" as the canonical name for it)

Reply:
well... for now there are no support to reading `.npmrc` files within workspaces

Reply:
that seems like something worth doing

Reply:
or at least considered, might be waaay more work than it sounds though and probably rfc-worthy